components/sekhras/route.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
```

Comment:
why is this a separate model from the Sekhra??  are they shared? Comment your code.